### DISCUSSION TITLE
Copilot not working

### DISCUSSION THREAD

**Discussion by nibea006** [community/community#120713] [Copilot Conversations]
### Select Topic Area

Question

### Body

Im trying to get github copilot on my computer to work. I got it to work once, but i dont know what i did. It says that i have to update my visual studio, but i cant. I know that it worked once on the current version i have and wondered if anyone could tell me how?

![Skjermbilde 2024-04-22 132959](https://github.com/community/community/assets/166801442/8298e22c-afc3-4a76-8063-8c32f526b956)

**Comment by eymeen** [comment]
have you tried to update the extension?

**Reply by nibea006** [reply]
it dosent display the new extension version. Is there a way to update it any other way?

![image](https://github.com/community/community/assets/166801442/7c5656e2-5c94-4a8f-8b8d-3e8aa9ce8995)

**Reply by eymeen** [reply]
did it work?
